src/content/docs/reference/configuration.mdx

```diff
@@ -264,44 +221,45 @@ Options that influence the rules of a single group. Biome supports the following
 - style: Rules enforcing a consistent and idiomatic way of writing your code.
 - suspicious: Rules that detect code that is likely to be incorrect or useless.
 
+Each group can accept, as a value, a string that represents the severity or an object where each rule can be configured.
 
-### `linter.rules.[group].recommended`
-
-Enables the recommended rules for a single group.
-
-Example:
-
-
+When passing the severity, you can control the severity emitted by all the rules that belong to a group.
+For example, you can configure the `a11y` group to emit information diagnostics:
 
 ```json title="biome.json"
 {
   "linter": {
-    "enabled": true,
     "rules": {
-      "nursery": {
-        "recommended": true
-      }
+      "a11y": "info"
     }
   }
 }
 ```
 
+Here's the accepted values:
+- `"on"`: each rule that belongs to the group will emit a diagnostic with the default severity of the rule. Refer to the documentation of the rule, or use the `explain` command:
```

Comment:
I am not very fond of this wording, it doesn't feel very clear. If you have a better suggestion, please offer me one :)

Reply:
It is? That's great! It's just that it took me a while to write it, so I wasn't sure about the final product